### PR TITLE
CCS-36556: added config loaderOptions

### PR DESCRIPTION
**Pull request by vmukhin17**
Added config loaderOptions for staticAssetLoader

### CODE REVIEW

File: lib/staticAssetLoader/index.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ module.exports = {
 
     // TODO: think about allowing some of the loader options to be set via the
```
Comment (robbytx) [inline lib/staticAssetLoader/index.js:93]: Seems like we can remove this comment?

**Comment by robbytx** [comment]
@vmukhin17 can you give an example of the configuration option(s) that you expect to pass?

**Comment by vmukhin17** [comment]
We need to increase default timeout from 10000 to 30000 -https://github.com/bazaarvoice/firebird/pull/6478/files#diff-2cf140d836ad55960d0acabc32583229R36

**Comment by robbytx** [comment]
Apparently, the [2.3.0 release commit](https://github.com/bazaarvoice/bv-ui-core/tree/v2.3.0) was never pushed into master. 🤷‍♂️

So I've rebased the commit from this PR (30cfebd) onto that one, and force-pushed master, making the commit for this PR into: https://github.com/bazaarvoice/bv-ui-core/commit/74cf5ceda366d403640ba4281c079934e487ecc4

**Comment by robbytx** [comment]
Ok... this change was released as a minor version, since it brings new functionality.  It's now released and available: https://www.npmjs.com/package/bv-ui-core/v/2.4.0

**Pull request merged**